lightning/src/ln/channelmanager.rs

```diff
@@ -10959,6 +10969,10 @@ This indicates a bug inside LDK. Please report this error at https://github.com/
 			}
 		};
 		self.fail_holding_cell_htlcs(htlcs_to_fail, msg.channel_id, counterparty_node_id);
+		for (static_invoice, reply_path) in static_invoices {
+			let res = self.flow.enqueue_held_htlc_available(&static_invoice, HeldHtlcReplyPath::ToCounterparty { path: reply_path });
```

Comment:
Mentioned offline: I think this can create a race if the receiver is online. The receiver will release immediately, the release reaches the sender lsp, the sender lsp tries to release the htlc, but the htlc hasn't been added to `pending_intercepted_htlcs` because the timer tick hasn't fired yet. The release fails.

Reply:
More context: this race actually happened in an LDK Node test... so the sender retrying `held_htlc_available` (or holding off sending it initially, somehow) is probably higher priority than we thought.